docker-compose.yaml

```diff
@@ -51,10 +53,48 @@ services:
     environment:
       - SPARK_MASTER_URL=spark://spark-master:7077
 
+  minio:
+    image: minio/minio
+    container_name: spark-minio
+    expose:
+      - "9000"
+    ports:
+      - "9000:9000"
+      # MinIO Console is available at http://localhost:9001
+      - "9001:9001"
+    environment:
+      MINIO_ROOT_USER: minio
+      MINIO_ROOT_PASSWORD: minio123
+      MINIO_ACCESS_KEY: minio
+      MINIO_SECRET_KEY: minio123
+    healthcheck:
+      test: timeout 5s bash -c ':> /dev/tcp/127.0.0.1/9000' || exit 1
+      interval: 1s
+      timeout: 10s
+      retries: 5
+    # Note there is no bucket by default
+    command: server /data --console-address ":9001"
+
+  minio-create-bucket:
```

Comment:
Seems like nothing depends on this?

Reply:
I don't think we need it. Just local testing environment.

Reply:
You mean you're going to delete it?

Reply:
I mean i don't think we need anything to depends on it.

Reply:
Seems like the notebook should if it's expecting a bucket to exist

Reply:
👍

Reply:
:+1: